cmd/utils.go

```diff
@@ -229,3 +231,23 @@ func toS3ETag(etag string) string {
 
 	return etag
 }
+
+// NewCustomHTTPTransport returns a new http configuration
+// used while communicating with the cloud backends.
+// This sets the value for MaxIdleConns from 2 (go default) to
+// 100.
+func NewCustomHTTPTransport() http.RoundTripper {
```

Comment:
These functions suppose to be in `cmd/gateway/gateway.go`?

Reply:
Yeah we can move them, i had some issue.

Reply:
I can't moved since rootCAs: point to globalCAs, i think its fine to keep it at top level since its quite useful for even used at top-level.